src/coreclr/vm/ClrEtwAll.man

```diff
@@ -1775,6 +1775,30 @@
                         </UserData>
                     </template>
 
+                    <template tid="WaitStart_V1">
+                        <data name="ClrInstanceID" inType="win:UInt16" />
+                        <data name="LockID" inType="win:Pointer" />
+                        <data name="AssociatedObjectID" inType="win:Pointer" />
```

Comment:
The start event includes the same properties as `ContentionStart`: `ClrInstanceID`, `LockID`, `AssociatedObjectId` but no `ContentionFlags` or `LockOwnerThreadID`

Reply:
Is `ClrInstanceId` a legacy thing? It looks like it's not implemented on linux https://github.com/dotnet/runtime/blob/6f30b7500ea2c0d0fb2923ea1fc5ed3c4ce2f29c/src/coreclr/vm/threads.cpp#L1030-L1039

Reply:
> Is ClrInstanceId a legacy thing?

I suspect it was added in CLR 4.0 when support for in-proc SxS was added. Its still part of all our event definitions largely out of convention or the possibility that we'd do SxS again.